CHANGELOG.rst

```diff
@@ -5,6 +5,135 @@ amazon.aws Release Notes
 .. contents:: Topics
 
 
+v5.0.0
+======
+
+Release Summary
+---------------
+
+In this release we promoted many community modules to Red Hat supported status. Those modules have been moved from the commuity.aws to amazon.aws collection. This release also brings some new features, bugfixes, breaking changes and deprecated features. The amazon.aws collection has dropped support for ``botocore<1.21.0`` and ``boto3<1.18.0``.
```

Comment:
We should also call out dropping support for Ansible Core < 2.11

Reply:
Right! Added, thanks!